community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java

```diff
@@ -102,6 +101,8 @@ public Factory( String key, String... altKeys )
         /** Release all locks. */
         void releaseAll();
 
+        void markForTermination();
```

Comment:
No strong opinion about the name here. `markForTermination` is just consistent with `KernelTransaction#markForTermination()`.